src/js/edu-benefits/1990-rjsf/components/IntroductionPage.jsx

```diff
@@ -61,26 +65,31 @@ class IntroductionPage extends React.Component {
             </li>
           </ol>
         </div>
-        <div className="row progress-box progress-box-schemaform form-progress-buttons schemaform-buttons">
-          <div className="small-6 medium-5 columns">
-            <a href="/education/apply-for-education-benefits/">
-              <button className="usa-button-outline">« Back</button>
-            </a>
-          </div>
-          <div className="small-6 medium-5 end columns">
-            <ProgressButton
-              onButtonClick={this.goForward}
-              buttonText="Continue"
-              buttonClass="usa-button-primary"
-              afterText="»"/>
-          </div>
-        </div>
-        <div className="omb-info--container">
+        <SaveInProgressIntro
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}>
+          Complete the form before submitting to apply for education benefits with the 22-1990.
```

Comment:
It looks like our language here and in the HCA form differ, should we confirm the content here?

Reply:
Great point. We should probably just keep it like this for now and make another ticket for Peggy to go over in more detail.

Reply:
According to @peggygannon, we should use the simpler "Please complete the XXXX form to apply for education benefits" form. So we can switch to that here (I updated the 1990e already).